### DISCUSSION TITLE
Prisma connection/studio is too slow

### DISCUSSION THREAD

**Discussion by turkerdev** [prisma/prisma#9078] [Q&A]
Im working on node.js
My every connection to prisma/studio takes 1-2 mins.
I dont have any query issues but connection.
This makes development very slow, everytime i add new code i have to restart the process and wait prisma to connect for 1-2 mins.
In production environment it connects in a sec.

**Comment by ryands17** [comment]
@bajosiaa 👋 

That shouldn't be the case. Are you using a local database or hosted?

**Reply by turkerdev** [reply]
Looks like the problem is mongodb.
As soon as i started using postgres it connects immediately and studio works fine too.

**Reply by ryands17** [reply]
In that case, I would suggest opening an issue [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) so that we can look into why this is slow and fix it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
